bin/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$STORYBOOK" = "true" ] ; then
+  echo ">> Build Storybook"
+  exec yarn build-storybook
+else
+  echo ">> Build WebApp (Next.JS)"
+  exec yarn build
```

Comment:
Yarn build is done by default no ? 
Doesn't that build it twice ?

Reply:
Nop. :-p 

If you see in the Scalingo Documentation, you can see a warning red block with: "If the scalingo-postbuild hook is defined, the build step will not run."

https://doc.scalingo.com/languages/nodejs/start